koku/masu/external/kafka_msg_handler.py

```diff
@@ -287,6 +302,7 @@ def extract_payload(url, request_id, context={}):  # noqa: C901
 
     """
     temp_dir, temp_file_path, temp_file = download_payload(request_id, url, context)
+    clean_invalid_csv(temp_dir)
```

Comment:
I think your going to have to call this after `extract_payload_contents` since `download_payload` will result in the tarball being in the `temp_dir`